addons/website/static/src/components/editor/editor.scss

```diff
@@ -11,12 +11,16 @@
 }
 // Duplicate code from web_editor/wysiwyg_snippets.scss
 .editor_has_snippets {
+    overflow: hidden;
     > .oe_overlay.ui-draggable {
         .o_handles {
             display: none;
         }
     }
 }
+.o_website_editor iframe {
```

Comment:
targets the iframe with more precision so as not to impact any iframe added by a user

Reply:
What do you mean? In their Odoo modification or in their website? Because this style doesn't impact the website they're building. Any code that's inside the iframe is independent of the backend code. Including the code that the website builder will add inside the iframe.